### DISCUSSION TITLE
IBEX - Antigen Retrieval and Secondary Staining for FFPE Tissues

### DISCUSSION THREAD

**Discussion by macandcheese0815** [IBEXImagingCommunity/ibex_imaging_knowledge_base#212] [Q&A] [answered]
Hello everyone! 

I am new to IBEX and have a few questions about IBEX with FFPE tissues! 

_Some background for my questions:_ 
Our lab works with high-risk pathogens - the formalin fixation therefore also serves the purpose of deactivating our pathogens, which would then allow us to bring our tissues out of the containment facility for more downstream processes, which means using frozen tissues is not an option for us. 

I am recently trying to perform IBEX on these FFPE tissue samples. Upon literature review, I have learned that tyramide signal amplifcation (i.e. OPAL multiplex) is commonly used to achieve good staining of FFPE tissue samples, as stated in several papers (Radtke et al., 2020; Quidgley et al., 2023). 

Before committing to actually purchasing the reagents needed performing Opal multiplex, my supervisor wanted me to test out primary staining to see what it would look like without the more costly and rigorous secondary staining / TSA - and as expected, there is no real signal for the markers, and there was a lot of autofluorescence (which I also learned is a common challenge since aldehyde fixation often creates autofluorescent byproducts that cause lots of background).

Here are some of my questions: 

1. Although this may be rare, I was wondering if anyone has ever tried coupling the AR6 / AR9 antigen retrieval buffers with primary staining. I used citrate buffer for my test run w primary Abs staining, and it did not work, and I am curious if improving the antigen retrieval alone could potentially make the staining work. 

2. Knowing that the Opal kits are very well-suited for staining of FFPE tissues, I believe that Opal-multiplex would be the most reasonable next-step. Nevertheless, I want to know if anyone here knows of an alternative to doing TSA with Opal (since the reagents could be quite pricey!). **Has anyone here had success with staining FFPE tissues with unconjugated primary Abs, then with fluorescently-tagged secondary Abs - and apply LiBH4 for fluorophore inactivation to enable further staining rounds ?** Could this be a potential approach? 

Thank you so much in advance to anyone who takes the time to read this!! Any feedback or suggestions would be appreciated!!! 

<img width="564" alt="image" src="https://github.com/user-attachments/assets/86b16109-b99e-4e80-98dc-84dcf792676b">

**Comment by radtkea** [comment, marked as answer]
Hi Macandcheese0815:)

Welcome!!! We're so glad you are here. Thank you for asking your questions in the open.

What a cool application! Have you been in contact with [Dr. Emily Speranza](https://www.lerner.ccf.org/florida/speranza/)? Emily established Opal-plex, the use of Opal dyes with the IBEX dye inactivation protocol, to examine samples from high containment facilities. She is working on a protocol for our [JoVE collection](https://app.jove.com/methods-collections/3182) entitled, "High-throughput IBEX with use of staining automation for core facilities." 

Answers to your questions:
1. Yes, we have used dual antigen retrieval! We actually recommend it. Here is a longer discussion [thread](https://github.com/IBEXImagingCommunity/ibex_imaging_knowledge_base/discussions/57). We set up a program on the Leica Bond using ER1 and ER2. If you don't have access to this device, then you could use the buffers in a hot water bath or use a pressure cooker. Improving antigen retrieval could definitely improve antibody performance. Please see this discussion [thread](https://github.com/IBEXImagingCommunity/ibex_imaging_knowledge_base/discussions/153).
2. Yes, we have used unconjugated primary antibodies, followed by fluorescently conjugated secondary antibodies, to amplify signal in IBEX experiments. The only caveat is that antibodies are not stripped in IBEX experiments (See [Figure S2](https://www.pnas.org/doi/full/10.1073/pnas.2018488117)). Therefore, you have to be very careful in designing your experiments. 
3. TSA-based amplification is a good thought. There are alternatives to Opal dyes and I have shared your post with my friend/colleague. She will reach out and share her experiences.
4. For FFPE tissues, sometimes other strategies are needed to reduce autofluorescence before immunolabeling. We have had success with photo-irradiation. My colleagues have a protocol they could probably share with you. I just need to ask. We discuss photo-irradiation [here](https://github.com/IBEXImagingCommunity/ibex_imaging_knowledge_base/discussions/44).

Questions for you:
1. Are you working on human bone marrow samples? 
2. What antibodies are you using for CD4 and CD45? These [CD4](https://ibeximagingcommunity.github.io/ibex_imaging_knowledge_base/supporting_material/CD4_Unconjugated/0000-0003-4379-8967.html) and [CD45](https://ibeximagingcommunity.github.io/ibex_imaging_knowledge_base/supporting_material/CD45_AF700/0000-0003-4379-8967.html) antibodies worked for us in other human FFPE tissues. FFPE bone marrow samples have a lot of auto, unfortunately. When in doubt, start with a high concentration of antibody, e.g., 1:25-1:50. Always best to see a signal and then titrate. Just to be clear, the multi-clonal (2B11 + PD7/26) for CD45 from Novus Biologicals is the one that works for FFPE. Be careful about what conjugate you use based on [Yuri's](https://github.com/IBEXImagingCommunity/ibex_imaging_knowledge_base/discussions/139) experience. Note she does dual antigen retrieval too!

Best of luck! 
A

**Reply by macandcheese0815** [reply]
Hi Andrea,

Thank you so so much for your detailed and through response! 

I figured that one of the problems I might have was not using an antigen buffer with a high pH because some papers that used the two antibodies I had used a pH 9-10 buffer while I used citrate buffer (pH 6). Thank you for linking the threads about Ag retreival! I will make sure to check them out. 

For bullet point 2 - I was thinking of incorporating an antibody stripping step prior to another round of staining if that's the approach I am using. I think I saw in some papers that the AR6/AR9 buffers coupled with microwave treatment can achieve that! Perhaps that would help remove the caveat?

I will also make sure to check out the thread to reduce autorluorescence before labelling! 

Answers to the questions:
1. I am working with humanized mice lung tissues - so it's mouse tissues but we look for human markers. 
2. Antibodies I used were: hCD4 (AF647; clone RPA-T4 - ThermoFisher 51-0049-42) and hCD45 (AF532, HI30 - ThermoFisher 58-0459-42) 

Thank you again for all your advices! I will make sure to incorporate these suggestions in my next test!!

**Reply by radtkea** [reply]
Of course! Happy that I could help and you are so very welcome!

Antibody stripping: Yes, that's definitely possible. I have tried to avoid it because it can impact tissue integrity and antibody labeling performance in subsequent rounds. I also experienced more autofluorescence when I used the microwave to strip antibodies. Caveat: I didn't spend a lot of time investigating this approach but I found it hard to control the temperature/prevent the buffer from boiling. Hopefully you have a better workflow. I hope we can get you to a point where you can use direct conjugates and/or a indirect labeling for a few targets. This will streamline the process.

Very cool! Now, I _see_ that I am looking at the lung in the picture you provided:)
Do you have a list of markers that you would like to include? I could help you find good antibody clones to use. It seems you would want/need: structural markers for mouse targets that work in FFPE tissues and immune markers for human targets in FFPE tissue. I haven't gotten those 2 clones to work in human FFPE tissues, I don't believe.

The clones I shared with you are the best candidates for those 2 targets.

Best of luck!
A

**Reply by macandcheese0815** [reply]
Thanks Andrea for your detailed response! 

Yes I agree that using direct conjugates will be the most ideal approach! Once my next batch of tissue sections are ready I will test out a revised protocol incorporating the suggestions you and DasEFX130 have provided and I will update it here (hopefully it will be successful images!)  

You are right I would need mouse structural markers like alpha smooth muscle actin (SMA) and CD31. My project looks at different memory T cells, so I would need markers like human CD45RO, CD62L, CD25, CD69, and in the future I would also want to explore staining for transcription factors like FoxP3, STAT2 etc so that I can also differentiate the different lineages (altho TF staining would be a later goal after I get my staining for surface markers to work first!) My test stains only involve CD4 and CD45, really appreciate the clones you shared for FFPE! 

Thank you again!

**Comment by DasEFX130** [comment]
Hi Macandcheese0815 and Andrea,
Hope you are doing well. It looks like you trying a lot to optimize. Really nice thoughts Andrea—I appreciated reading the details! A few thoughts/questions:
1. Have you used a white light protocol to reduce endogenous autofluorescence by any chance? method described in [Nat Cancer. 2023 Jul;4(7):1036-1052](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10368530/)
2. As Andrea mentioned, Thermo has an Opal alternative kit that is coming out and the product manager could send you a full size kit to try if you would be up for it.  I could connect you.
Good luck! :) Thanks for letting us contribute too! :)

**Reply by radtkea** [reply]
Thank you so much for sharing this great paper and telling us about an alternative to Opal. Very exciting! Of course, we are delighted when people share their expertise with the community!

**Reply by macandcheese0815** [reply]
Hi DasEFX130,

Thank you so much for your response as well!! 

1. I haven't! I will look into that, thank you for linking such a useful paper! 
2. That is great news! I would love to get connected and learn more about this Thermo alternative! Having more options is awesome!

**Reply by DasEFX130** [reply]
Hey MacandCheese0815 and Andrea,
Happy Monday.
1. Hope it helps! :)
2. Edward Parkins (mailto:edward.parkin@thermofisher.com) (hope that linked correctly) could help with it. :)
Good luck!

**Reply by macandcheese0815** [reply]
Thank you so much DasEFX130!! Really appreciate it.
